src/main/java/com/stripe/Stripe.java

```diff
@@ -10,6 +14,7 @@ public abstract class Stripe {
   public static final int DEFAULT_READ_TIMEOUT = 80 * 1000;
 
   public static final String API_VERSION = ApiVersion.CURRENT;
+  public static final String PREVIEW_API_VERSION = ApiVersion.PREVIEW_CURRENT;
```

Comment:
I'm a little wary about setting this as a public property on `Stripe`, but otherwise `PREVIEW_CURRENT` isn't accessible from `LiveStripeResponseGetter` and tests in the `net` package. Plus this is in the beta branch anyways, where users can expect volatility.